file_paths.md

```diff
@@ -59,6 +59,11 @@ The files annotated by an '*' indicate that they are created by package installa
         routes.yaml                       # :route_file
         ssl                               # :ssldir                      /etc/puppet/ssl
 
+    /etc/puppetlabs/pxp-agent *
+        modules.d *                       # directory where pxp-agent modules are stored
+            puppet.cfg *                  # /opt/puppetlabs/pxp-agent/modules/puppet configuration
+        pxp-agent.cfg                     # pxp-agent configuration file
```

Comment:
Yep, that makes sense. So yeah, keep the two config files. 

I'd suggest the module one here should be called `pxp-module-puppet.conf`.